src/main/kotlin/io/csbroker/apiserver/auth/LoginUserArgumentResolver.kt

```diff
@@ -29,12 +24,10 @@ class LoginUserArgumentResolver : HandlerMethodArgumentResolver {
         webRequest: NativeWebRequest,
         binderFactory: WebDataBinderFactory?
     ): Any? {
-        var principal: Any? = null
-
         val authentication = SecurityContextHolder.getContext().authentication
 
-        if (authentication != null) {
-            principal = authentication.principal
+        val principal = authentication?.let {
+            authentication.principal
         }
 
         if (authentication == null || principal is String) {
```

Comment:
```suggestion
        return SecurityContextHolder
            .getContext()
            .authentication
            ?.principal ?: throw UnAuthorizedException(ErrorCode.UNAUTHORIZED, "로그인이 필요합니다.")
```

`authentication`이 null일때와 `principle`이 String일때를 각각 다르게 에러 핸들링을 하는게 아니면 일괄적으로 method chaining & null check만 해주는게 어떨까요?
일단 `AuthTokenProvider`를 확인해봤을 때 `principal`에 String이 들어가는 경우가 있을까 싶은데 이 부분은 제가 잘 모르는 부분이라 String Type check를 꼭 해주어야 하나요??

Reply:
저도 왜 해줬는지 기억이 안나지만.. 필요 없을 것 같아 제거했습니다.